src/components/atoms/markdown.tsx

```diff
@@ -67,14 +67,15 @@ const MarkdownLink = (
         };
 
     return screenReaderEnabled ? (
-      <TouchableWithoutFeedback
+      <TouchableOpacity
```

Comment:
Since we're using `TouchableOpacity` for the tab selectability not for the opacity can we add `activeOpacity={1}` with a comment that we can't use `TouchableWithoutFeedback` because it can't be selected by keyboard tab?